src/handleRuntimeProblems.js

```diff
@@ -1,4 +1,4 @@
-const blueprintUtils = require('./blueprint-utils');
+const blueprintUtils = require('./blueprintUtils');
```

Comment:
Maybe it would be better to:

```js
const { warningLocationToRanges, rangesToLinesText } = require('./blueprintUtils');
```

Not gonna comment on the whole "utils" module :).

Reply:
Once I get to rewrite this to use source maps with line numbers, I think I can get rid of the whole module.